### ISSUE TITLE
MvxRecyclerView crashes on .NET 8

### ISSUE THREAD

**Issue by evgenyvalavin** [MvvmCross/MvvmCross#4778]
## 🐛 Bug Report
Set .NET 8 as the target framework for an Android project.
Run the app. Get the crash:
```
"System.NotSupportedException: Could not activate JNI Handle 0x7fd8ca4680 (key_handle 0xe4784bd) of Java type 'mvvmcross/droidx/recyclerview/MvxRecyclerView' as managed type 'MvvmCross.DroidX.RecyclerView.MvxRecyclerView'.
 ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> MvvmCross.Exceptions.MvxException: Unable to find resource type. Please check if your setup class is in your application's root namespace.
   at MvvmCross.Platforms.Android.Binding.ResourceHelpers.MvxAppResourceTypeFinder.Find()
   at MvvmCross.DroidX.RecyclerView.AttributeHelpers.MvxRecyclerViewAttributeExtensions.TryInitializeBindingResourcePaths()
   at MvvmCross.DroidX.RecyclerView.AttributeHelpers.MvxRecyclerViewAttributeExtensions.ReadRecyclerViewItemTemplateSelectorClassName(Context context, IAttributeSet attrs)
   at MvvmCross.DroidX.RecyclerView.AttributeHelpers.MvxRecyclerViewAttributeExtensions.BuildItemTemplateSelector(Context context, IAttributeSet attrs, Int32 itemTemplateId)
   at MvvmCross.DroidX.RecyclerView.MvxRecyclerView..ctor(Context context, IAttributeSet attrs, Int32 defStyle, IMvxRecyclerAdapter adapter)
   at MvvmCross.DroidX.RecyclerView.MvxRecyclerView..ctor(Context context, IAttributeSet attrs)
   at System.Reflection.MethodBaseInvoker.InterpretedInvoke_Constructor(Object obj, IntPtr* args)
   at System.Reflection.MethodBaseInvoker.InvokeConstructorWithoutAlloc(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
```
[Changes in Android resource generation in .NET 8](https://devblogs.microsoft.com/dotnet/android-resource-designer-dotnet-8/)

### Expected behavior
No crash.

### Reproduction steps
1. Create a .NET 8 Android project.
2. Add an MvxRecyclerView
3. Run app

**Version:** 9.1.1

**Platform:** 
- [ ] :robot: Android

**Comment by Cheesebaron** [comment]
This is fixed in 9.2.0 and is due to how resources are now generated in net8.0-android.

**Comment by evgenyvalavin** [comment]
Great news!
Forgot to look for the closed issues...
